src/content/blog/new-domain-new-discord-same-project.md

```diff
@@ -0,0 +1,60 @@
+---
+title: New Domain, new Discord, same Project!
+description: .xyz is out, .dev is the new black.
+publishDate: 2024-11-27
+ogVariant: winter
+author: louis-escher
+---
+
+Hey all!
+
+Although it's been a while since we last talked, I can assure you that work on StudioCMS has been moving forward. We've been making good progress on 
+`beta.8` and hope to release it towards the end of 2024, although that's more of a guess than a prediction.
+
+You've read the title (or so I hope), so what am I on about here?
+
+Well, you've no doubt seen that we have been [sponsored by Turso](https://studiocms.dev/blog/turso-sponsorship), which were and still are incredibly 
+grateful for. However, we're trying to improve our experience as much as possible for both new users and contributors to the project. At this point 
+we feel we can move away from Astrolicious, the community organization who's been helping us nurture our little project, and go in alone to have full 
+control over our own ecosystem! After talking to [Florian](https://github.com/florian-lefebvre) and [Alex](https://github.com/alexanderniebuhr), 
+we've come to the conclusion that it's time for a fresh coat of new paint. Therefore, I introduce to you:
+
+## The new StudioCMS Discord Server
+That's right, we've escaped the confines of our single channel on Astrolicious' Server and made ourselves [a new home](https://chat.studiocms.dev). This 
+will be the place for you to chat and receive news about StudioCMS from now on! *Don't worry, the blog isn't going anywhere ;)*
+
+Moving to the new Discord Server has already allowed us to do some pretty cool things. For example, yours truly has been hard at work at our new Open 
+Source Discord Bot, [Apollo](https://github.com/withstudiocms/apollo)! If you're on the Astro Discord, you'll know what they're good at. PTAL notifications, 
+moderation tools (soon™️), support forum utilities, it's all there.
+
+You can even use this Bot in your own Servers! All you need is a GitHub App, Discord App, and a libSQL database connection, for which we obviously use 
+and highly recommend [Turso](https://turso.tech/). You can take a look at the [README file](https://github.com/withstudiocms/apollo?tab=readme-ov-file#apollo) 
+to learn more.
+
+But if you really **did** read the title (or clicked one of the links in this article), you might've also noticed another thing. Thus, it is with great 
+pleasure that I announce... *Drumroll please...* 🥁
+
+## The new StudioCMS GitHub Org
```

Comment:
```suggestion
## The new StudioCMS GitHub organization
```

Reply:
Same thing as earlier, Chicago Manual of Style. If anything I'd do "Organization"